types/index.d.ts

```diff
@@ -18,6 +18,7 @@ export interface SolrServiceOptions extends ServiceOptions {
   matcher?: (query: any) => any;
   sorter?: (sort: any) => any;
   Model: FetchClient | UndiciClient;
+  escapeFn: (field: string, value: any) => any
```

Comment:
Because we use an return Object with the properties `key` and `value` in the `lib/core/query`, we should define in here the return value more explicit than any.